src/app.module.ts

```diff
@@ -1,9 +1,18 @@
 import { Module } from '@nestjs/common';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
+import { UsersModule } from './users/users.module';
+import { TypeOrmModule } from "@nestjs/typeorm";
+import { ConfigModule } from "@nestjs/config";
 
 @Module({
-  imports: [],
+  imports: [
+      // Domyślnie Nest nie czyta envów, trzeba zadeklarować poniżej, jakie pliki mają być użyte i w jakiej kolejności
+      ConfigModule.forRoot({envFilePath: ['.env.local', '.env']}),
+      UsersModule,
+      TypeOrmModule.forRoot(
+        { type: "postgres", host: process.env.DATABASE_HOST, port: parseInt(process.env.DATABASE_PORT), username: process.env.DATABASE_USER, password: process.env.DATABASE_PASSWORD, database: process.env.DATABASE_NAME, synchronize: true, entities: ["**/*.entity.js"] }
```

Comment:
Guess we will use `.ts` extensions 😬 

```suggestion
        { type: "postgres", host: process.env.DATABASE_HOST, port: parseInt(process.env.DATABASE_PORT), username: process.env.DATABASE_USER, password: process.env.DATABASE_PASSWORD, database: process.env.DATABASE_NAME, synchronize: true, entities: ["**/*.entity.ts"] }
```

Reply:
`.js` is accurate because it is configuration for compiled files in `dist` folder. However I saw that nestjs now provides `autoLoadEntities: true` params so I will check this up.

Reply:
good to know! tnx! :D